pages/common/rg.md

```diff
@@ -0,0 +1,23 @@
+# ripgrep
+
+> A fast command-line search tool.
+
+- Recursively search the current directory for a regex pattern:
+
+`rg {{pattern}}`
+
+- Search for pattern including all .gitignored and hidden files:
+
+`rg -uu {{pattern}}`
+
+- Search for a pattern only in a certain filetype (e.g., html, css, etc.):
+
+`rg -t {{filetype}} {{pattern}}`
+
+- Search for a pattern only in a subset of directories:
+
+`rg {{pattern}} {{set_of_subdirs}}`
```

Comment:
Tokens can be either descriptive names (e.g. {{url}}) or generalized examples demonstrating the syntax, if that's more useful (e.g. {{http://example.com}}. In this case I feel the latter would be better. If you agree, you can make a new PR, or let me know and I'll do the change myself :)